src/Dom/Document.php

```diff
@@ -1021,6 +1023,34 @@ private function restore_doctype_node( $html ) {
 		return preg_replace( self::HTML_RESTORE_DOCTYPE_PATTERN, '\1!\3\4>', $html, 1 );
 	}
 
+	/**
+	 * Normalizes HTML attributes to be HTML5 compatible.
+	 *
+	 * Conditionally removes html[xmlns], and converts html[xml:lang] to html[lang].
+	 */
+	private function normalize_html_attributes() {
+		$html = $this->getElementsByTagName( 'html' )->item( 0 );
+		if ( ! $html->hasAttributes()  ) {
+			return;
+		}
+
+		$xmlns                = $html->attributes->getNamedItem( 'xmlns' );
+		$xmlns_value_to_strip = 'http://www.w3.org/1999/xhtml';
+		if ( $xmlns && $xmlns_value_to_strip === $xmlns->nodeValue ) {
+			$html->removeAttributeNode( $xmlns );
+		}
```

Comment:
Since the variable is only used once, might as well remove `$xmlns_value_to_strip` and just use the string literal:

```php
		if ( $xmlns && 'http://www.w3.org/1999/xhtml' === $xmlns->nodeValue ) {
```

Reply:
Yeah, that probably makes sense to remove the variable declaration.

Reply:
Applied in https://github.com/ampproject/amp-wp/pull/4194/commits/096cf7b880794ac352e9b0235a1eeb46a802f144